source/components/reusable/Icon/IconListContainer.js

```diff
@@ -29,8 +35,27 @@ export default class IconListContainer extends Component {
       .catch( componentErrorHandler( 'IconListContainer' ) )
   }
 
+  turnIntoProject() {
+    const newProjectName = this.props.text
+
+    this.deleteItems()
+    axios.post( `${__HOST__}/project/new`, { text: newProjectName })
+      .then( ( newProject ) => {
```

Comment:
Since the only part of `newProject` we care about here is `id`, we can descructure it out:
```js
.then( ({ data: { id }) ) => {
```